OPENWRT.md

```diff
@@ -189,6 +188,59 @@ Type following command to boot from usb.
 
     $ pistachio # run usbboot
 
+## Boot from SD Card
+Similar to the instructions for Boot from usb drive, you need to format the SD card into ext4 parition. Follow the steps mentioned above by making subtle changes
+
+
+`$ mount | grep /media
+/dev/sdc1 on /media/nikhil.zinjurde/5c0dac14-1d47-4a39-8b06-e27861473670 type ext4 (rw,nosuid,nodev,uhelper=udisks2)'
+
+SD media essentially mounts at /dev/sdc1 on a computer.  However, make sure to double-check if it gets somehow mounted differently on your computer, in which case, <b>IMPORTANT! Make sure that you replace /dev/sdc with /dev/sdx for the right device name</b>
+
+Unmount the filesystem before creating new partition table:
+
+    $ sudo umount /dev/sdc1
+
+Create new partition table:
+
+    $ sudo fdisk /dev/sdx
+    > enter 'o' to create a new empty DOS partition table
+    > enter 'n' to create a new partition; just press return at every prompt to accept the default values
+    > enter 'w' to write the new partition table and exit
+    $ sudo mkfs.ext4 /dev/sdx1
+
+Mount the SD media:
+
+    $ sudo mount /dev/sdc1 /mnt/
+
+To put filesystem on SD you will need
+
+    openwrt-pistachio-pistachio_marduk-uImage
```

Comment:
since now we have uImage and dtb available in the rootfs, should we mention this?
while you are on this, should we remove from usbboot too?